project/Dependencies.scala

```diff
@@ -58,6 +58,7 @@ object Dependencies {
       val sprayJson   = Compile.sprayJson                                                                    % "test"
       val gson        = "com.google.code.gson"             % "gson"                    % "2.8.2"             % "test"
       val jacksonXml  = "com.fasterxml.jackson.dataformat" % "jackson-dataformat-xml"  % jacksonVersion      % "test" // ApacheV2
+      val reflections = "org.reflections"                  % "reflections"             % "0.9.11"            % "test" // WTFPL
```

Comment:
Weird that doesn't resolve for me:

```
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/org/reflections/reflections/0.9.11/reflections-0.9.11.${packaging.type}
[warn] ==== jcenter: tried
[warn]   https://jcenter.bintray.com/org/reflections/reflections/0.9.11/reflections-0.9.11.${packaging.type}
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::              FAILED DOWNLOADS            ::
[warn] 	:: ^ see resolution messages for details  ^ ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.reflections#reflections;0.9.11!reflections.${packaging.type}
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

Reply:
https://github.com/ronmamo/reflections/issues/169

Reply:
it worked quite nicely for me, and for Jenkins / Travis

Reply:
Hmm... seems it was an issue in the pom that was later fixed by republishing the same artifact. For some reason I've got the old one in my ivy cache from a year ago. After cleaning that entry it works :+1:

Reply:
Cool!